src/App.jsx

```diff
@@ -1,35 +1,31 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react";
+import reactLogo from "./assets/react.svg";
+import viteLogo from "/vite.svg";
+import "./App.css";
+import Form from './Form';
 
 function App() {
-  const [count, setCount] = useState(0)
+	const [showForm, setShowForm] = useState(false);
 
-  return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
-  )
+	const handleShow = () => {
+		setShowForm(true);
+	};
+
+	const handleHide = () => {
+		setShowForm(false);
+	};
+
+	const form = showForm ? <Form /> : null;
```

Comment:
PART 1: Passing a prop

This is where you are running into issues with hiding the form on submit. This is where props come into play. The only way to get data to move from the child to the parent is via callback function.

Using the `toggleForm` function from my other comment:
```suggestion
	const form = showForm ? <Form handleToggleForm={toggleForm} /> : null;
```
Take a look at your `Form` component to see how you would hide on submit